kuma/wiki/templates/wiki/document.html

```diff
@@ -172,7 +172,8 @@
             <a class="button from-search-next only-icon disabled" title="{{ _('Next Search Result') }}" data-empty-title="{{ _('No Previous Search Result') }}"><i aria-hidden="true" class="icon-chevron-right"></i></a>
           </span>
 
-        <h1>{{ document.title }}</h1>
+        {% set safe_title = document.title|forceescape %}
+        <h1>{{ safe_title|addbreaks|safe }}</h1>
```

Comment:
@groovecoder Please don't just assign it to you or ask dev-webdev. Auto escaping is a core ability of kuma and should be handled by its developers. In other words, this is too serious to do in between.